### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by Evan-CTL**

### CODE REVIEW

**Comment by nikolas** [comment]
These changes include a committed .DS_Store file - this should be removed and ignored in your global .gitignore. Also - the history here should be cleaned up with a rebase.

**Comment by nikolas** [comment]
Also - the changes updating the compiled JS files at `viewer.js` and `editor.js` to your personal link files need to be reverted here.

**Comment by Evan-CTL** [comment]
Hopefully that's the last of it.

**Comment by nikolas** [comment]
Getting closer, but this branch has 12 commits on it - these should all be squashed down to a single commit with a concise description.

**Comment by Evan-CTL** [comment]
It was the -f tag. All of that nonsense and apparently if you don't hit use the -f tag to force push the rebase to the remote repo it won't squash the remote commits in the PR. UUGH!!! That at up the past day and a half. I must have rebased all of those commits at least 5 times.

**Comment by nikolas** [comment]
Hah - no worries! Soon you'll know git like the back of your hand - it takes a while to get used to.

**Comment by nikolas** [comment]
I just updated the commit message with `git commit --amend`, then updated this branch with `git push origin accessibility -f` - you can see the new joint commit here on GitHub. A lot of the stuff like flake8 fixes and .DS_Store stuff doesn't need to be mentioned in the final commit message, as it's not really relevant to the actual accessibility changes being made here.

The actual changes look good though, and I think I can finally merge this!

File: econplayground/templates/base.html [hunk 1]
```diff
@@ -68,9 +68,9 @@
 
 {% user_is_instructor request.user as i_am_instructor %}
 <body class="{% block bodyclass %}{% endblock %}" id="{% block bodyid %}{% endblock %}">
-    <div id="skip">
-        <a href="#maincontent">Skip to main content</a>
-    </div>
+    <aside id="skip">
+        <a href="/accounts/login#maincontent">Skip to main content</a>
```
Comment (nikolas) [inline econplayground/templates/base.html:72]: Actually - I'm not sure if this was intentional here. This causes the "Skip to main content" button, selectable when you press "Tab" on any page of the site, to always redirect to the login form, even when you're already logged in. I think this should be changed back to the original #maincontent href, unless I'm missing something.

**Pull request merged**